tests/test_tasks.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import json
+import os
+from data_quality import tasks, utilities, compat
+from goodtables import pipeline
+
+class TestTasks(unittest.TestCase):
+    """Test `tasks.py` """
+
+    def test_aggregator_run(self):
```

Comment:
Please give some feedback on these tests. They seem more like functional than unit tests either because these "units" have a lot of functionality or because I'm doing something wrong.